DiffCalculatorActor/DiffCalculatorActor.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Actors;
+using Microsoft.ServiceFabric.Actors.Runtime;
+using Microsoft.ServiceFabric.Actors.Client;
+using DiffCalculatorActor.Interfaces;
+using Common.Model;
+
+namespace DiffCalculatorActor
+{
+    /// <remarks>
+    /// This class represents an actor.
+    /// Every ActorID maps to an instance of this class.
+    /// The StatePersistence attribute determines persistence and replication of actor state:
+    ///  - Persisted: State is written to disk and replicated.
+    ///  - Volatile: State is kept in memory only and replicated.
+    ///  - None: State is kept in memory only and not replicated.
+    /// </remarks>
+    [StatePersistence(StatePersistence.Persisted)]
+    internal class DiffCalculatorActor : Actor, IDiffCalculatorActor
+    {
+        private DeviceReading _previousReading;

```

Comment:
I like it, but it's wrong 👯‍♂️ 

Je moet je previous reading ook in realiable state opslaan. Anders wanneer SF zou rebalancen of het systeem herstart, ga je previous readings missen.

https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-actors-state-management

Check deze link even hoe je gebruik maakt van reliable state.

deze state wordt ook netjes over meerdere nodes gepersisteerd.